spec/lib/facilities/drivetime_bands/response_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'facilities/ppms/response'
```

Comment:
Why is the ppms response being required here?

Reply:
Good catch, definitely not needed 👍